account_banking_pain_base/models/account_payment_mode.py

```diff
@@ -33,3 +33,8 @@ class AccountPaymentMode(models.Model):
         "the PAIN files generated by Odoo. This value is determined by the "
         "financial institution that will process the file. If not defined, "
         "no scheme will be used.\n")
+
+    default_batch_booking = fields.Boolean(
+        string='Default Batch Booking',
+        help="Default value for payment order batch booking related "
+             "to this payment mode")
```

Comment:
Don't you need to add the field to the view too?

Reply:
yes